services/core/functions/demo-review-lambda/index.ts

```diff
@@ -40,6 +52,13 @@ export const main = async (event: APIGatewayProxyEvent) => {
       });
     }
 
+    const demoReviewDayCounterEntity =
+      getDemoReviewDayCounterEntity(TABLE_NAME);
```

Comment:
I think entities should abstract out which table is being used from the function code. This is why I would prefer it set in the entity if possible.

Reply:
So how would we set it in the entity ? Reconstruct the table name ?